apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -1095,7 +1140,9 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     /**
     * @dev Copy settings from one storage pointer to another
     */
+   // Note: should we inline?
     function _copySettings(Setting storage _from, Setting storage _to) private {
+        // Note: we may want to check if solidity is smart enough to not require us to first load into local vars to optimize sstores
```

Comment:
Similar note to above; these should be loadable (and re-storable) in 1-2 SLOADs, but since we've chunked them like this, I'm not sure the compiler is smart enough :)

Reply:
I'm not exactly sure what would be the alternative here

Reply:
I would double check with remix to see what the compiler does here, but the ugly alternative is to first load them all into local vars and then push them back into storage. So something ugly like:

```
uint256 supportRequiredPct_ = from.supportRequiredPct;
uint256 minAcceptQuorumPct_ = from.minAcceptQuorumPct;
...

to.supportRequiredPct = uint64(supportRequiredPct_);
to.minAcceptQuorumPct = uint64(minAcceptQuorumPct_);
...
```

Reply:
I tried this optimization, but it makes the contract hit the bytecode size limit... not sure if that's a good tradeoff at the end if we need to sacrifice the cost of the rest of the functions

Reply:
I checked this removing some functions and it actually increases the costs haha